server/chat-commands/room-settings.ts

```diff
@@ -854,12 +855,13 @@ export const commands: ChatCommands = {
 
 	deletechatroom: 'deleteroom',
 	deletegroupchat: 'deleteroom',
+	dgc: 'deleteroom',
 	deleteroom(target, room, user, connection, cmd) {
 		room = this.requireRoom();
 		const roomid = target.trim();
 		if (!roomid) {
 			// allow deleting personal rooms without typing out the room name
-			if (!room.settings.isPersonal || cmd !== "deletegroupchat") {
+			if (!room.settings.isPersonal || cmd !== "deletegroupchat" && cmd !== "dgc") {
```

Comment:
```suggestion
			if (!room.settings.isPersonal || !['deletegroupchat', 'dgc'].includes(cmd)) {
```

I believe this helps readability, if only because you're no longer mixing `&&` and `||`.